README.md

```diff
@@ -138,14 +140,11 @@ That could be used out of the box in `knp_paginator.template.pagination` key:
 That could be used out of the box in `knp_paginator.template.sortable` key:
 
 * `@KnpPaginator/Pagination/sortable_link.html.twig` (by default)
-* `@KnpPaginator/Pagination/bootstrap_v5_bi_sortable_link.html.twig`
-* `@KnpPaginator/Pagination/bootstrap_v5_fa_sortable_link.html.twig`
-* `@KnpPaginator/Pagination/bootstrap_v5_md_sortable_link.html.twig`
-* `@KnpPaginator/Pagination/twitter_bootstrap_v3_sortable_link.html.twig`
-* `@KnpPaginator/Pagination/twitter_bootstrap_v4_font_awesome_sortable_link.html.twig`
-* `@KnpPaginator/Pagination/twitter_bootstrap_v4_material_design_icons_sortable_link.html.twig`
-* `@KnpPaginator/Pagination/semantic_ui_sortable_link.html.twig`
-* `@KnpPaginator/Pagination/uikit_v3_sortable.html.twig`
+
```

Comment:
Please restore removed templates from the list

Reply:
Ooop, I didn't mean to remove those! I've reinstated them, and also put my new section above them, as it relates directly to pagination rather than sorting or filtering.

Reply:
I have added some usage notes to the readme.